md/tenant_admin_credentials.md

```diff
@@ -21,9 +21,9 @@ For an already existing tenant, you can change these tenant-specific credentials
 
 
 ## Important notice (only for default tenant)
-The engine and the portal can be installed on different machines / Tomcat / WildFly installations. For that reason, the portal has its own configuration file
+The engine and the portal can be installed on different machines / Tomcat installations. For that reason, the portal has its own configuration file
 that define the tenant technical user credentials, in order to communicate with the (potentially remote) engine.
 
 For this matter, if you change the tenant admin credentials (default tenant only) on engine side (see above), you must also update the corresponding values in the file:
 `<WEBSERVER_HOME>/setup/platform_conf/`**initial**`/platform_portal/platform-tenant-config.properties` if you have never started the platform yet, and
-`<WEBSERVER_HOME>/setup/platform_conf/`**current**`/platform_portal/platform-tenant-config.properties` if the platform has been already started.
\ No newline at end of file
+`<WEBSERVER_HOME>/setup/platform_conf/`**current**`/platform_portal/platform-tenant-config.properties` if the platform has been already started.
```

Comment:
Why WEBSERVER_HOME ? we always say bundle...
It applies as well to previous lines